src/components/App/index.jsx

```diff
@@ -0,0 +1,75 @@
+import React from 'react';
+import { Switch, Router } from 'react-router-dom';
+
+import { Helmet } from 'react-helmet';
+import { AuthenticatedPageRoute, PageRoute, AppProvider } from '@edx/frontend-platform/react';
+
+import { getAuthenticatedHttpClient } from '@edx/frontend-platform/auth';
```

Comment:
minor suggestion: group imports together, e.g. this @edx could go with the one above, in-project imports can be one group next and so on

Reply:
I typically do (third party on top, external edx imports next, then in-project imports)

Reply:
Agree, I usually do that too. I think we should make the linter enforce it, though, because otherwise I forget. (And it takes up code review time) :)

Reply:
yeah linter is a good idea.. can check